FBTweak/_FBTweakCollectionViewController.m

```diff
@@ -64,6 +77,24 @@ - (void)viewWillDisappear:(BOOL)animated
   [_keyboardManager disable];
 }
 
+- (BOOL)_supportsUpdates {
+  return [self.tweakCategory respondsToSelector:@selector(updateWithCompletion:)];
+}
+
+- (void)_updateCategory {
+  self.refreshControl.attributedTitle =
+      [[NSAttributedString alloc] initWithString:@"Refreshing..."];
+  [self.refreshControl beginRefreshing];
```

Comment:
Do we want to call it on main thread as well?

Reply:
This method is called on the main thread - it's the observer for the refresh control.